refinery/ui/source/js/data-set-visualization/ctrls/visualization-ctrl.js

```diff
@@ -57,6 +58,15 @@
         });
     }
 
+    function deleteTool (vis) {
```

Comment:
It'd be nice to add some comments. 
ex:     /**
     * Generates sort param for api call from ui-grid response and calls grid
     * reset
     * @param {obj} grid - ui-grid obj
     */